packages/preview/lab-report-template-ustc/1.0.0/content/template/main.typ

```diff
@@ -0,0 +1,50 @@
+#import "../../theme.typ": *
```

Comment:
I'm not sure what the role of this file is?

Reply:
I think somebody may not want to edit example.typ directly, so I provide an empty template for them.

Reply:
The thing is, this file will be quite hard to discover for users of your package: only the `content/example` folder will be copied to their project, so unless they dig in the repository or in the local package cache (which most people don't do I would say), they will never see it.

In the end it is up to you to decide whether you want to keep these files or not, but I given how hard they would be to find I'm not sure they add much. Also, it is relatively easy to select all or part of the example file and delete it, if someone wants a more minimal template.

Reply:
Since I often modify the templates I use to meet my specific style requirements, I believe there will still be people checking the repository. 
In the final version I submitted, I removed unnecessary files, even though they were quite small.